TrafficCapture/dockerSolution/src/main/docker/elasticsearchTestConsole/simpleDocumentQuery.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python
+import requests
+from datetime import datetime
+import argparse
+import os
+import urllib3
+import time
+from collections import deque
+from concurrent import futures
+import logging
+
+# Suppress only the single InsecureRequestWarning from urllib3 needed for this script
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+# Set the base URLs from the environment variables or use default values
+source_url_base = os.getenv('SOURCE_DOMAIN_ENDPOINT', 'https://elasticsearch:9200')
+target_url_base = os.getenv('MIGRATION_DOMAIN_ENDPOINT', 'https://opensearchtarget:9200')
+source_username = 'admin'
+source_password = 'admin'
+target_username = 'admin'
+target_password = 'myStrongPassword123!'
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--source-endpoint",
+        help="Source cluster endpoint e.g. http://source.elb.us-west-2.amazonaws.com:9200."
+    )
+    parser.add_argument(
+        "--target-endpoint",
+        help="Target cluster endpoint e.g. http://target.elb.us-west-2.amazonaws.com:9200."
+    )
+    parser.add_argument(
+        "--source-username",
+        help="Source cluster username.",
+        default=source_username
+    )
+    parser.add_argument(
+        "--source-password",
+        help="Source cluster password.",
+        default=source_password
+    )
+    parser.add_argument(
+        "--target-username",
+        help="Target cluster username.",
+        default=target_username
+    )
+    parser.add_argument(
+        "--target-password",
+        help="Target cluster password.",
+        default=target_password
+    )
+    parser.add_argument(
+        "--source-no-auth",
+        action='store_true',
+        help="Flag to provide no auth in source requests."
+    )
+    parser.add_argument(
+        "--target-no-auth",
+        action='store_true',
+        help="Flag to provide no auth in target requests."
+    )
+    parser.add_argument(
+        "--no-clear-output",
+        action='store_true',
+        help="Flag to not clear the output before each run. Helpful for piping to a file or other utility."
+    )
+    return parser.parse_args()
+
+
+def get_latest_document(session, url_base, auth):
+    url = f"{url_base}/simple_doc_*/_search"
+    query = {
+        "size": 1,
+        "sort": [
+            {
+                "timestamp": {
+                    "order": "desc"
+                }
+            }
+        ]
+    }
+    try:
+        response = session.get(url, json=query, auth=auth, verify=False, timeout=0.5)
+        response.raise_for_status()
+        hits = response.json().get('hits', {}).get('hits', [])
+        if hits:
+            latest_doc = hits[0]['_source']
+            return latest_doc
+        else:
+            return None
```

Comment:
When does this happen? Worth adding a log message?

Reply:
This happens when no test docs exist on the source/target cluster. This will happen before we start RFS/Replayer for the target.

We propogate the None through and print out N/A for the relevant logs